### PR TITLE
fix mb_substr missing parameter error when generating japanese string with realText method

### PR DESCRIPTION
**Pull request by HejunweiCoder**
fix mb_substr missing parameter error when generating japanese string with realText method

### CODE REVIEW

**Comment by HejunweiCoder** [comment]
may be this problem happened in php7

File: src/Faker/Provider/ja_JP/Text.php [hunk 1]
```diff
@@ -620,7 +620,7 @@ protected static function appendEnd($text)
     {
         // extract the last char of $text
         if (function_exists('mb_substr')) {
-            $last = mb_substr($text, mb_strlen($text)-1, null, 'UTF-8');
+            $last = mb_substr($text, 0, mb_strlen($text) - 1, null, 'UTF-8');
```
Comment (fzaninotto) [inline src/Faker/Provider/ja_JP/Text.php]: I don't understand. `mb_substr()`'s signature has only 4 arguments.

http://php.net/manual/en/function.mb-substr.php

**Comment by fzaninotto** [comment]
Thanks!

**Comment by HejunweiCoder** [comment]
You are welcome,I forget to remove that 'null' parameter...@fzaninotto

**Pull request merged**